project.clj

```diff
@@ -55,10 +55,14 @@
                  [org.postgresql/postgresql "9.2-1003-jdbc4"]
                  [clojureql "1.0.3"]
                  ;; MQ connectivity
-                 [clamq/clamq-activemq "0.4" :exclusions [org.slf4j/slf4j-api]]
-                 [org.apache.activemq/activemq-core "5.7.0" :exclusions [org.slf4j/slf4j-api org.fusesource.fuse-extra/fusemq-leveldb]]
+                 [org.apache.activemq/activemq-core "5.7.0"
+                  :exclusions [org.slf4j/slf4j-api
+                               org.fusesource.fuse-extra/fusemq-leveldb]]
+                 [org.apache.activemq/activemq-pool "5.7.0"
+                  :exclusions [org.slf4j/slf4j-api]]
+
                  ;; bridge to allow some spring/activemq stuff to log over slf4j
```

Comment:
@mullr They were when I made these changes, but I'll be checking again before the final version.